addons/sale_timesheet/models/project.py

```diff
@@ -211,9 +211,13 @@ def write(self, values):
             })
         return res
 
+    def _possibly_updated_timesheets(self):
```

Comment:
We could name it `_get_timesheets` (we have the same method in `project.task` for instance) or `_get_updatable_timesheets` for instance. 🙂